integrationtests.py

```diff
@@ -24,7 +24,7 @@
 
 PYTHON_BINARY = sys.executable
 CLCACHE_SCRIPT = os.path.join(os.path.dirname(os.path.realpath(__file__)), "clcache.py")
-ASSETS_DIR = os.path.join("tests", "integrationtests")
+ASSETS_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), "tests", "integrationtests")
```

Comment:
This was changed only to simplify the code in TestBasedir. The cwd is changed with `cd()` and the relative path to the assets wouldn't work. If I rearrange the code there a bit this change is not needed.